ee/modules/110-istio/hooks/dataplane_metadata_exporter.go

```diff
@@ -26,6 +27,7 @@ const (
 	istioVersionUnknown          = "unknown"
 	istioPodMetadataMetricName   = "d8_istio_dataplane_metadata"
 	metadataExporterMetricsGroup = "metadata"
+	telemetryGroup               = "istio_telemetry"
```

Comment:
A suggestion: start all telemetry metrics with `telemetry_*`, or use the `d8_istio_*` pattern